gradle-language-server/src/main/java/com/microsoft/gradle/GradleServices.java

```diff
@@ -131,7 +131,17 @@ public void didChangeWatchedFiles(DidChangeWatchedFilesParams params) {
   @Override
   public void didChangeConfiguration(DidChangeConfigurationParams params) {
     Map<?, ?> settings = new Gson().fromJson((JsonElement) params.getSettings(), Map.class);
-    LSPUtils.applySetting(this, settings);
+    this.applySetting(this, settings);
+  }
+
+  public void applySetting(GradleServices services, Object settings) {
```

Comment:
why do you need to pass a `GradleService`? Can it be `public void applySetting(Object settings)` ?